bundles/core/org.eclipse.smarthome.core.persistence/src/main/java/org/eclipse/smarthome/core/persistence/PersistenceAdminService.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.persistence;
+
+import org.eclipse.smarthome.core.items.Item;
+
+/**
+ * Provides an administration interface to allow the system to purge data from the persistence store.
+ *
+ * @author Chris Jackson - Initial implementation and API
+ *
+ */
+public interface PersistenceAdminService {
+    /**
+     * Returns a list of items that are stored in the persistence service
```

Comment:
Ok, in this case the interface name was very misleading. It should simply also be a XXXPersistenceService, shouldn't it? So as discussed above -> combine those into a single interface.